tests/setup_environment_test/setup_environment_test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
```

Comment:
Could you also add instructions for how to run these tests - ideally as a wiki page. This could note that GH actions is setup and they will run there when pushed, and it can be run locally using `python -m pytest` (and `python -m unittest` once renamed) in the root directory. It'll also provide a nice framework for Pooja to make additional wiki pages.